content/using-sagemaker-managed-spot-training/notebooks/running_notebooks.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Running Notebooks"
+chapter: false
+weight: 30
+---
+
+1. Within JupyterLab (Which you opened in the previous steps), navigate to the ***/ec2-spot-labs/ec2-spot-sagemaker-managed-spot-training/*** folder.
+
+2. Double-click on a Notebook and complete each step outlined in the Notebook. Each available lab comes in the form of a separate Notebook. Locate the labs you are interested in running and complete each step documented in the notebook.  ![SageMaker Notebook Instance](/images/using-sagemaker-managed-spot-training/jupyter-3.png)
```

Comment:
is there an assumption that workshop participants are familiar with using notebooks? how about a getting started link just in case?

Reply:
Yeah that's a good call. We'll probably have some attendees that have never touched Jupyter. Let me clean this up a bit.